platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/client/impl/SecureCxfClientFactoryImpl.java

```
@@ -570,6 +570,10 @@ private void configureConduit(ClientConfiguration clientConfig) {
       tlsParams.setUseHttpsURLConnectionDefaultSslSocketFactory(false);
       tlsParams.setCertAlias(keyInfo.getAlias());
       try {
+        if (keyManagers == null) {
```

❓ Do we want to throw an exception if keyManagers is null?